### DISCUSSION TITLE
Welcome to cryptopump Discussions!

### DISCUSSION THREAD

**Discussion by aleibovici** [aleibovici/cryptopump#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NathBish** [comment]
For someone who came from the hardware side of tech. What would be the easiest way to get cryptopump working? I did have a read you need to install a few things first etc. Any chance of a prebuilt system? What sort of hardware is required and what accounts do I need to setup?

**Reply by aleibovici** [reply]
NathBish, right now, we do not have a pre-built system, like in a docker container. So that is a roadmap item.

To get it working, the easiest way is to install Go (https://golang.org/dl/) and download the latest package release (https://github.com/aleibovici/cryptopump/releases/tag/v1.5).

You will also need a MySQL, but I recommend running on Docker (https://hub.docker.com/_/mysql)

Make sure you configure your environment variables  DB_USER, DB_PASS, DB_TCP_HOST, DB_PORT, DB_NAME and also the config.yml file, following instructions from https://github.com/aleibovici/cryptopump

To run your code: go run main.go

I like to run in Microsoft Visual Studio code, but it is not necessary.
